### PR TITLE
[BugFix] Fix digest inconsistency caused by in predicate child number

### PR DESCRIPTION
**Pull request by zenoyang**
## What I'm doing:

Fixes #issue

## What type of PR is this:

- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [x] Yes, this PR will result in a change in behavior.
- [ ] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.3
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Comment by zenoyang** [comment]
@murphyatwork Please review the code, thank you.

File: fe/fe-core/src/main/java/com/starrocks/sql/common/SqlDigestBuilder.java [hunk 1]
```diff
@@ -93,5 +93,17 @@ public String visitLimitElement(LimitElement node, Void context) {
             sb.append(" ? ");
             return sb.toString();
         }
+
+        @Override
+        public String visitInPredicate(InPredicate node, Void context) {
```
Comment (murphyatwork) [inline fe/fe-core/src/main/java/com/starrocks/sql/common/SqlDigestBuilder.java:98]: what's the difference between above `visitInPredicate` ?

**Comment by murphyatwork** [comment]
I guess it's been fixed by https://github.com/StarRocks/starrocks/pull/52578

**Comment by zenoyang** [comment]
> I guess it's been fixed by #52578

Yes, my PR will be closed soon.😂

**Pull request closed without merging**